File: C++/Iniciante/1038.cpp
#include<iomanip>
#include<iostream>

using namespace std;

int main(){
	int cod, qnt;
	double total;
	cin >> cod >> qnt;
	switch(cod){
		case 1:
			total=qnt*4;
			break;
		case 2:
			total=qnt*4.50;
			break;
		case 3:
			total=qnt*5;
			break;
		case 4:
			total=qnt*2;
			break;
		case 5:
			total=qnt*1.50;
			break;
	}		
			
	cout << fixed << setprecision(2);
	cout << "Total: R$ " << total << endl;
	return 0;
}

File: Python3/Iniciante/1038.py
# -*- encoding: utf-8 -*-

N = input()

COD = int(N.split()[0])
Q = int(N.split()[1])

if (COD == 1):
    print("Total: R$ %.2f" % (Q*4))
elif (COD == 2):
    print("Total: R$ %.2f" % (Q*4.5))
elif (COD == 3):
    print("Total: R$ %.2f" % (Q*5))
elif (COD == 4):
    print("Total: R$ %.2f" % (Q*2))
elif (COD == 5):
    print("Total: R$ %.2f" % (Q*1.5))
